scripts/build-stories.js

```diff
@@ -138,6 +138,7 @@ async function buildStories() {
     } catch (e) {
       console.error(e);
       process.exit(-1);
+      continue;
```

Comment:
This line should probaby be removed as well.

Reply:
Yeap, removed